dts/riscv/riscv32-litex-vexriscv.dtsi

```diff
@@ -63,5 +63,14 @@
 			label = "eth0";
 			status = "disabled";
 		};
+		i2c0: i2c@e0003000 {
+			compatible = "litex,i2c";
+			reg = <0xe0003000L 0x3 0xe0003004L 0x1>;
```

Comment:
We've historically not use'd 'L' in the hex values.  So having this just be `<0xe0003000 0x3 0xe0003004 0x1>

Reply:
Changed.